learning_journal/templates/edit-entry.jinja2

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
```

Comment:
You **must not** put anything into a jinja2 template before the `{% extends "blah.jinja2" %}` line.  That has to be the first thing in the file.  Your `boostatmenu.jinja2` template should be providing a doctype tag.